egs/sprakbanken/s5/local/chain/compare_wer_general.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+echo $0 $*
+
+echo -n "System               "
+for x in $*; do   printf "% 10s" " $(basename $x)";   done
```

Comment:
Change $* to "$@"

It's important in case someone has an experimental directory that contains a space. That's probably never going to happen, but you never know.